src/query/ast/src/parser/expr.rs

```diff
@@ -658,11 +658,11 @@ pub fn expr_element(i: Input) -> IResult<WithSpan<ExprElement>> {
     let position = map(
         rule! {
             POSITION
-            ~ ^"("
-            ~ ^#subexpr(BETWEEN_PREC)
-            ~ ^IN
-            ~ ^#subexpr(0)
-            ~ ^")"
+            ~ "("
```

Comment:
please add some comments to help other people understand the difference.

Reply:
Ok that's a draft and I think we can modify more in this pr.